specs/binding-http.spec/src/main/resources/META-INF/zilla/http.idl

```diff
@@ -49,4 +49,77 @@ scope http
             HttpHeader[] headers;
         }
     }
+
+    scope event
+    {
+        enum HttpEventType (uint8)
+        {
+            CORS_PREFLIGHT_REQUEST_NOT_ALLOWED (1),
+            CORS_REQUEST_NOT_ALLOWED (2),
+            GUARD_AUTHORIZATION_FAILURE (3),
+            GUARD_AUTHORIZATION_SUCCESS (4),
+            ROUTE_NOT_FOUND (5),
+            ROUTE_FOUND (6)
+        }
+
+        enum Level
+        {
+            INFO,
+            WARNING,
+            ERROR
+        }
+
+        struct HttpEventCommon
+        {
+            int64 timestamp = 0;
+            int64 originId;
+            int64 routedId;
+            int64 initialId;
+            int64 replyId;
+            int64 affinity;
+            int64 authorization = 0;
+            int64 budgetId;
+            int64 traceId;
+        }
```

Comment:
Perhaps this is just a generic `Event` defined in `engine` that can be reused across different `.idl` files?

Also, probably just need `timestamp` through `replyId` and `traceId`, not `affinity`, `authorization` or `budgetId`.

It's also worth taking a non-binding example next, since there is no stream concept there for context during logging.

Reply:
refactored this to introducing Event and BindingEvent to core.idl, pls chk

Reply:
what would be a good example for creating log events in something that's not a binding? like JwtGuardHandler.reauthorize? 🤔 pls LMK